pkg/apis/submariner.io/v1/endpoint.go

```diff
@@ -198,3 +199,19 @@ func (ep *EndpointSpec) ParseSubnets(family k8snet.IPFamily) []net.IPNet {
 
 	return subnets
 }
+
+func (ep *EndpointSpec) ExtractSubnets(family k8snet.IPFamily) []string {
```

Comment:
So this method internally uses `GetPrivateIP`, which is hidden to the user. Perhaps this is specific to the libreswan driver but, either way, I suggest we make it explicit by naming it `ExtractPrivateIPSubnets` or generalize it and pass in the IP, ie `ExtractSubnets(forIP string, family k8snet.IPFamily)`.